Makefile

```diff
@@ -0,0 +1,5 @@
+build-image:
+	docker build . -t 541497436480.dkr.ecr.eu-west-1.amazonaws.com/protoc
+
+push-image:
+	docker push 541497436480.dkr.ecr.eu-west-1.amazonaws.com/protoc:latest
```

Comment:
if this project expects to see more updates, it's best to avoid the `latest` tag and use a git-rev hash derived ID. see https://github.com/danielqsj/kafka_exporter/blob/master/Makefile#L8 , for example.